src/index.js

```diff
@@ -63,7 +63,7 @@ export default async function educandu(options) {
     }
 
     if (serverConfig.initialUser) {
-      const existingUser = await userService.getActiveUserByEmailAddress(serverConfig.initialUser.email);
+      const existingUser = await userService.getActiveUserByEmail(serverConfig.initialUser.email);
```

Comment:
We could also here use `getActiveConfirmedUserByEmail`, right? The initial user should be "usable", or is there something speaking against it?

Reply:
Well if the user does not exist, we create it. But a non confirmed user is still an existing user and inserting into the database would fail.
Of course, it would not fail in this particular case because we create the test user (unless created somehow by hand beforehand), but I think the code would be confusing if we'd user `getActiveConfirmedUserByEmail`.

Reply:
Oh, it's the only usage you say 🤔 Well in this case I can live with reducing the helper methods, easily 😄 
Changed!